tests/stubtest_third_party.py

```diff
@@ -43,7 +43,15 @@ def run_stubtest(dist: Path, *, verbose: bool = False, specified_stubs_only: boo
 
     with tempfile.TemporaryDirectory() as tmp:
         venv_dir = Path(tmp)
-        venv.create(venv_dir, with_pip=True, clear=True)
+        try:
+            venv.create(venv_dir, with_pip=True, clear=True)
+        except subprocess.CalledProcessError as e:
```

Comment:
Maybe we should only suppress the traceback if it's the known issue with `ensurepip`:

```suggestion
        except subprocess.CalledProcessError as e:
            if "ensurepip" not in e.cmd:
                raise
```